.flake8

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
We've got flake8 config in `setup.cfg` too, let's pick one config style and stick with it.

Reply:
Totally. Moved the parameters to `setup.cfg` thereby making `.flake8` redundant.